distributed/tests/test_scheduler.py

```diff
@@ -1609,6 +1609,36 @@ def test_dashboard_address():
         assert not s.workers
 
 
+@gen_cluster(client=True)
+def test_net_nbytes(c, s, a, b):
+    dsk = {
+        "a-1": (bytearray, 1000),
+        "a-2": (bytearray, 1000),
+        "b-2": (lambda t, u: t[:10], "a-1", "a-2"),
+    }
+    x = c.get(dsk, "b-2", sync=False)
+    result = yield x
+    assert result == bytearray(10)
+    assert s.task_net_nbytes["a"] > 0
+    assert s.task_net_nbytes["b"] < 0
+
+    # now that we've learned some things, submit again, and
+    # check the workers.
```

Comment:
So we're not going to learn this on initial large computations, correct?

Reply:
That's unfortunately correct. Once (if) the initial large computation completes we'll think that other tasks with the same prefix are likely to be large.

I see two possible solutions

1. (Long term) task annotations could store the information we need, avoiding the need to learn for the initial computation.
2. (Short term) Provide an API for users to inform the scheduler, though I don't think this is a good idea.